<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>viewdoc</class>
 <widget class="QDialog" name="viewdoc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>700</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 117, 143);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbllenspre1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <property name="text">
    <string>Lens Preset One</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="linedocid">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbllenspre2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>340</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <property name="text">
    <string>Lens Preset Two</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbllenspre3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>420</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <property name="text">
    <string>Lens Preset Three</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbllensformula">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>500</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <property name="text">
    <string>Primary Formula</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblpatid">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="linedocname">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblpatname">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="linedocpre1">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>260</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="linedocpre2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>340</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="linedocpre3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>420</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="linedocformula">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>500</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblpatname_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(222, 221, 218);
font: 75 20pt &quot;Ubuntu&quot;;
font:bold;</string>
   </property>
   <property name="text">
    <string>View Doctor</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="BUTOK">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>600</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>ok</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
